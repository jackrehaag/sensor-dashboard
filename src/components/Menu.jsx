import React from "react";

const Menu = props => {
  return (
    <div className="ui menu">
      <div className="header item">IOT Sensor Dashboard</div>
      <a href="#" className="item">
        Sensors
      </a>
      <a href="" className="item">
        Status
      </a>
    </div>
  );
};

export default Menu;
