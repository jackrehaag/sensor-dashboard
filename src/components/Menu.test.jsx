import React from "react";
import Menu from "./Menu";
import renderer from "react-test-renderer";

test("it renders", () => {
  const component = renderer.create(<Menu />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
