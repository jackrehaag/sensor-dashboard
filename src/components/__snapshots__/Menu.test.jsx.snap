// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<div
  className="ui menu"
>
  <div
    className="header item"
  >
    IOT Sensor Dashboard
  </div>
  <a
    className="item"
    href="#"
  >
    Sensors
  </a>
  <a
    className="item"
    href=""
  >
    Status
  </a>
</div>
`;
